server/src/routes/api/v1/saladsRouter.js

```diff
@@ -7,9 +7,15 @@ const saladsRouter = new express.Router()
 
 saladsRouter.get("/", async (req, res) => {
     try {
-        const salads = await Salad.query()
+        const saladsSansUsers = await Salad.query()
+        const salads = await Promise.all(saladsSansUsers.map( async (salad) => {
+            salad.user = await salad.$relatedQuery("user")
+            salad.user = salad.user.username
+            return salad
+        }))
         return res.status(200).json({ salads: salads })
     } catch (error) {
+        console.log(error)
```

Comment:
should remove console logs because this is going to return the error as the response (this console. log is great while you are developing and working with this route, but should be removed for final code)